services/rewards/handler/handler.go

```diff
@@ -0,0 +1,52 @@
+package handler
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/brave-intl/bat-go/libs/handlers"
+	"github.com/brave-intl/bat-go/libs/logging"
+	"github.com/brave-intl/bat-go/services/rewards"
+	"github.com/brave-intl/bat-go/services/rewards/model"
+)
+
+type cardService interface {
+	GetCardsAsBytes(ctx context.Context) (rewards.CardBytes, error)
+}
+
+type CardsHandler struct {
+	cardSvc cardService
+}
+
+func NewCardsHandler(cardSvc cardService) *CardsHandler {
+	return &CardsHandler{
+		cardSvc: cardSvc,
+	}
+}
+
+const errSomethingWentWrong model.Error = "something went wrong"
```

Comment:
nit: can we make this error a bit more clear about the issue so it can be handled on the FE properly?

Reply:
This error message is returned when we get a 500 i.e. a generic error that indicates the server encountered an unexpected condition and we cannot fulfil the request. We don't want to return the error "as is" because it could contain system information particularly when being returned by third party in this case S3. 

For a 500 there is not much the FE can do so we log the error in the backend to debug.

Reply:
Gotcha, makes sense to me. It wasn't clear to me this was a catchall for 500s so might be useful to name it more in line with that, but that's just a nit